go/general_style_guide.md

```diff
@@ -127,7 +127,19 @@ You can run `nancy` locally by building or downloading the binary and running `g
 
 Please don't push secrets to GitHub unless they are encrypted.
 
-We recommend [configuring a client-side git hook](https://github.com/zricethezav/gitleaks/wiki/Scanning#uncommitted-changes-scan) so that your secrets are never pushed in the first place.
+We recommend [configuring a client-side git hook](https://github.com/zricethezav/gitleaks/wiki/Scanning#uncommitted-changes-scan) so that your secrets are never pushed in the first place. To do this:
+```
+# Create a directory to hold your global git hooks, if you don't have one
+$ mkdir /path/to/your/git-hooks
+
+# Create a file named pre-commit in that directory, with the content from the gitleaks link above
```

Comment:
Can we add that in here directly? shouldn't be too hard to `touch` a file and then pipe that stuff in

Reply:
Yeah I can include it here, my thinking was that this might get out of date with upstream then